simulate-travis.py

```diff
@@ -257,6 +263,58 @@ def _set_channel_order():
     sp.check_call([bin_for('conda'), 'config', '--get'])
 
 
+def _travis_run(env):
+    if (
+        (env['TRAVIS_OS_NAME'] == 'linux') &
+        (not args.disable_docker) &
+        ('--docker' not in env['BIOCONDA_UTILS_BUILD_ARGS'])
+    ):
+        env['DOCKER_ARG'] = '--docker'
+        if not args.disable_mulled:
+            env['DOCKER_ARG'] += ' --mulled-test'
+
+    # Override env with whatever's in the shell environment
+    env.update(os.environ)
+
+    # Only modify the path just before running travis-run.sh:
+    if 'CONDA_ROOT' in os.environ:
+        env['PATH'] = os.path.join(
+            os.environ['CONDA_ROOT'], 'bin') + ':' + env['PATH']
+
+    try:
+        sp.check_call(['scripts/travis-run.sh'], env=env, universal_newlines=True)
+    except sp.CalledProcessError:
+        sys.exit(1)
+
+
+def _docker_machine_run_travis(env):
+    try:
+        p = sp.check_output(['docker-machine', 'env'])
+        os.environb.update({
```

Comment:
I suppose I've learned to distrust shell interaction using unicode, too many things that can unexpectedly go wrong because some language flag is set. In this case, it shouldn't matter, either way would work fine.

 The parsing is sort of a hack anyway. A steel wrought solution would evaluate the output in `bash` as it is meant to be used and dump the resulting environment in a controlled fashion. But I believe it is rather unlikely that the docker folks will ever put anything funny in that output.